cybsi/api/dictionary/api.py

```diff
@@ -360,8 +360,9 @@ class DictionaryItemForm(JsonObjectForm):
     Args:
         dictionary_uuid: Dictionary identifier.
         key: Dictionary item key. The key should consist of characters
-            according to the pattern `[a-zA-Z0-9_ -]` and
-            have length in the range `[1;30]`.
+            according to the pattern `[a-zA-Z0-9_ -:.@`] and
```

Comment:
кажется в 2.13 это снова изменится) но тут ок

Reply:
Это паттерн для регулярки, надо - (минус) последним символом ставить (или экранировать). Плюс квадратная скобка съехала
```suggestion
            according to the pattern `[a-zA-Z0-9_ :.@-]` and
```